modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/CheckboxTag.java

```diff
@@ -148,6 +176,12 @@ protected int processStartTag() throws Exception {
 			jspWriter.write(" disabled");
 		}
 
+		if (Validator.isNotNull(_id)) {
```

Comment:
Looking at how clay `Checkbox` component is working, you should render here `class="getCssClass()` (pseudocode) since component html attributes are being applied to the input, not to the container.

Reply:
@carloslancha did you comment this on the `id` section on purpose or was it meant to have a different context?

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/427/files#r510783145